public/typescript/gui.ts

```diff
@@ -56,6 +57,9 @@ export const handleServerResponse = (status: string) => {
   }
 };
 
+// tslint:disable-next-line:no-console
+export const log: (...args: any[]) => void = typeof console !== 'undefined' ? bind(console.info, console) : empty;
```

Comment:
Can you explain how, and when this log export is used? I do not see it used in this PR, at least not in a way I can recognize it.

Reply:
Something when wrong merging i think..